src/main/java/org/quantumbadger/redreader/fragments/PostListingFragment.java

```diff
@@ -812,6 +813,10 @@ public void onDataStreamComplete(
 									&& mPostListingURL.asSubredditPostListURL().type
 									== SubredditPostListURL.Type.SUBREDDIT);
 
+							final String keywordFilter = PrefsUtility.pref_behaviour_posts_filter();
+
+							final String[] keywordFilterArray = keywordFilter.split(",");
```

Comment:
The keywords should be trimmed of whitespace here. It would also be good to move the `toLowerCase` on line 851 here to avoid doing it for every post.